.github/workflows/external-issue-labeler.yml

```diff
@@ -0,0 +1,45 @@
+# SPDX-FileCopyrightText: Copyright (c) 2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: Triage outside issues
+
+on:
+  issues:
+    types:
+      - opened
+
+env:
+  GITHUB_TOKEN: ${{ secrets.ISSUE_PR_WRITE_GITHUB_TOKEN }}
+
+jobs:
+  Label-Issue:
+    runs-on: ubuntu-latest
+    # Only run if the issue author is not part of RAPIDS
+    if: ${{ ! contains(fromJSON('["OWNER", "MEMBER", "CONTRIBUTOR", "COLLABORATOR"]'), github.event.issue.author_association)}}
+    steps:
+      - name: add-external-labels
+        run: |
+          issue_url=${{ github.event.issue.html_url }}
+          gh issue edit ${issue_url} --add-label "Needs Triage,External"
+
+      - name: add-comment-to-issue
```

Comment:
Personally I am not in favor of autoresponders if their text is so simple. It adds noise to the conversation that I don't think is helpful. If it were "smart" and pointed to a contributing guide for the relevant component (libcudf/cudf Python) or linked to potentially related issues, then perhaps.

Reply:
What Slack integration are you talking about? I already get GitHub notifications via Slack for every new issue that is filed.

Reply:
Part of a new process to track/triage `External` issues easier. https://github.com/integrations/slack#creating-a-filter when issues with `External` exist they can go into specific channels to ensure timely response/handling.

The goal is to reduce noise of all issues/comments and only focus on a small subset.